### ISSUE TITLE
Fallback image / alt text when product card image is missing

### ISSUE THREAD

**Issue by austin-liminal** [limlabs/saleor-storefront-starter#74]
See the linked code for the use case where the product card image could be empty. In this case, we need to have:

- Localized alt text for "product image not found"
- A fallback / placeholder to indicate no product image was loaded
